source/tutorial/autocomplete-component.md

```diff
@@ -250,15 +250,15 @@ In the `filterByCity` function in the rentals controller above,
 we've added a new property called `query` to the filter results instead of just returning an array of rentals as before.
 
 ```app/components/list-filter.js{-19,-9,+10,+11,+12,+20,+21,+22,+23,+24}
-import Ember from 'ember';
+import Component from '@ember/component';
```

Comment:
I think the previous change was introduced with old style.
If you want to keep using `import Ember from 'ember';`, you need to modify the line below from `export default Component.extend({` to `export default Ember.Component.extend({`